### PR TITLE
Added plot themes

### PR DESCRIPTION
**Pull request by raviselker**

### CODE REVIEW

File: R/theme.R [hunk 1]
```diff
@@ -0,0 +1,36 @@
+theme <- new.env()
```
Comment (jonathon-love) [inline R/theme.R]: can you add some roxygen documentation for this?

and you should add `'# @export` too

File: R/themes.R [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        },
+        iheartspss = function() {
+            self$hadley + ggplot2::theme(
+                panel.border = ggplot2::element_rect(colour="#333333", fill=NA, size=0.5),
+                panel.grid.major = ggplot2::element_blank(),
+                panel.grid.minor = ggplot2::element_blank(),
+                legend.key = ggplot2::element_blank())
+        },
+        minimal = function() {
+            self$hadley + theme$hadley + ggplot2::theme(
```
Comment (jonathon-love) [inline R/themes.R]: well thars your problem!

File: R/themes.R [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                panel.background= ggplot2::element_rect(fill='transparent', color=NA),
+                axis.line = ggplot2::element_blank(),
+                panel.grid = ggplot2::element_blank(),
+                panel.grid.major = ggplot2::element_line(colour = '#E8E8E8'),
+                panel.grid.minor = ggplot2::element_blank(),
+                axis.ticks = ggplot2::element_blank(),
+                legend.key = ggplot2::element_blank())
+        },
+        liberace = function() {
+            self$hadley + theme$hadley + ggplot2::theme(
```
Comment (jonathon-love) [inline R/themes.R]: and there

File: R/themes.R (hunk 2 above)
Comment (raviselker) [inline R/themes.R]: whoops :)

File: R/themes.R [hunk 4]
```diff
@@ -0,0 +1,63 @@
+#' This contains the jamovi plot themes
+#'
+#' `hadley` is a theme based on the default ggplot2 theme
+#' `default` is the default theme in jamovi
+#' `iheartspss` is a theme based on the default SPSS theme
+#' `minimal` is a minimal theme
+#' `liberace` is an gold theme
```
Comment (jonathon-love) [inline R/themes.R]: `a gold theme`

Comment (jonathon-love) [inline R/themes.R]: you could make these bit more explicit with `themes$hadley`, `themes$default`, etc.

Comment (jonathon-love) [inline R/themes.R]: i also think the first line of the roxygen is the title. so `the jamovi plot themes` would probably be better than `This contains the jamovi plot themes`

File: R/themes.R [hunk 5]
```diff
@@ -0,0 +1,63 @@
+#' The jamovi plot themes
+#'
+#' This contains the following plot themes:
+#' `themes$default` is the default theme
+#' `themes$hadley` is a theme based on the default ggplot2 theme
+#' `themes$iheartspss` is a theme based on the default SPSS theme
+#' `themesminimal` is a minimal theme
```
Comment (jonathon-love) [inline R/themes.R]: missing a dollar sign in `themes$minimal`

**Pull request merged**